src/_P096_eInk.ino

```diff
@@ -279,7 +279,9 @@ boolean Plugin_096(byte function, struct EventStruct *event, String& string)
 
     case PLUGIN_WRITE:
       {
+#ifdef DEBUG_LOG
```

Comment:
We already have `BUILD_NO_DEBUG`
So please change into `#ifndef BUILD_NO_DEBUG`

Reply:
OK I made a mistake (I've searched in espeasy code, and found somewhere DEBUG_LOG, so I've decided to use it)
Fixed ;-)